level.go

```diff
@@ -1,18 +1,44 @@
 package engo
 
+import (
+	"engo.io/gl"
+)
+
 type Level struct {
-	Width      int
-	Height     int
+	width      int
+	height     int
 	TileWidth  int
 	TileHeight int
 	Tiles      []*tile
-	LineBounds []Line
+	LineBounds []*Line
 	Images     []*tile
 }
 
+func (t *tile) Height() float32 {
+	return float32(t.height)
+}
+
+func (t *tile) Width() float32 {
+	return float32(t.width)
+}
+
+func (t *tile) Texture() *gl.Texture {
+	return t.Image.Texture()
+}
+
+func (t *tile) Close() {
```

Comment:
You don't need to, I think in the end that's how it should be though. previous comment applies here too